cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,62 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const username = 'test_qa';
+  const password = 'Qwerty123@';
 
+  const book = {
+    title: 'Speaking JavaScript',
+    description: 'Like it or not, JavaScript is everywhere these days'
+  };
+
+  beforeEach(() => {
```

Comment:
You have to create custom login js command for log in with API and then use it in your tests